tools/codegen/core/gen_switch.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python2.7
```

Comment:
Can we try python3 here since we're on python3 migration path?

Reply:
Good idea, done.